gtfsmerger/__init__.py

```diff
@@ -2,6 +2,21 @@
 from gtfsmerger.gtfs import GTFS
 
 
+def post_process(func):
+    def wrapper(self, gtfs_obj):
+        merged = func(self, gtfs_obj)
+        if 'calendar_dates' in merged:
+            merged['calendar_dates'].loc[:, 'date'] = pd.to_datetime(merged[
+                'calendar_dates'].loc[:, 'date'].astype(str))
```

Comment:
Is there a reason you are first converting to string, before converting to datetime?